site/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,142 @@
+.grid.three {
```

Comment:
@amymanion ok, but i would create partials for these. they are unrelated.

for example,
* ` site/_scss/site/objects/_grid.scss`
*  `site/_scss/site/objects/_sub-page.scss`
*  `site/_scss/site/objects/_blog-card.scss`

that way each SCSS file is a bit more self-contained.